### PR TITLE
Created sidebars for navigation to different pages | Added style to m…

### PR DESCRIPTION
**Pull request by IvayloSlavchev**
…ain page in order to render components next to each other | Left sidebar is for navigation to different pages, here I use MUI icons instead of plain text | On the right side is story template, it's only static styles and text for now

### CODE REVIEW

**Review by petya0111 — COMMENTED** [review]
@IvayloSlavchev  You can move the styles for the bars in the main.scss to be reusable from the future pages.

**Review by petya0111 — APPROVED** [review]
Great work with SideBars. Thank you! ✓

File: client/src/components/Header.tsx [hunk 1]
```diff
@@ -4,48 +4,62 @@ import { useState } from "react";
...
-import { Spacings } from "../values/Spacings";
-import { Avatar, TextField } from "@mui/material";
+import { TextField } from "@mui/material";
 import ClearIcon from "@mui/icons-material/Clear";
 import SearchIcon from "@mui/icons-material/Search";
-import headerLogo from "/src/images/headerLogo.png";
+import headerLogo from '/src/images/logo-mark.svg'
+
+import AccountCircleRoundedIcon from '@mui/icons-material/AccountCircleRounded';
+import { Box } from '@mui/material';
```
Comment (cholakovvv) [inline client/src/components/Header.tsx]: This can be imported with TextField in line 7

File: client/src/components/NavigationBar/Navigation.tsx [hunk 2]
```diff
@@ -10,12 +10,11 @@ import LogoutOutlinedIcon from '@mui/icons-material/LogoutOutlined';
...
-
 export const NavigationBar = () => {
     let tokenMap: Map<number, boolean> = new Map();
 
     //I use any here because it throws an error when trying to set the type to Map<number, boolean>
-    const [mapState, setMapState]: any = React.useState(null);
+    const [mapState, setMapState]: any = useState(null);
 
 
     const IconArray: any = [
```
Comment (cholakovvv) [inline client/src/components/NavigationBar/Navigation.tsx:22]: Same here

File: client/src/components/NavigationBar/Navigation.tsx [hunk 3]
```diff
@@ -10,12 +10,11 @@ import LogoutOutlinedIcon from '@mui/icons-material/LogoutOutlined';
 //File will execute automatically
 import '../Styles/NavigationBar.scss';
 
-
 export const NavigationBar = () => {
     let tokenMap: Map<number, boolean> = new Map();
 
     //I use any here because it throws an error when trying to set the type to Map<number, boolean>
-    const [mapState, setMapState]: any = React.useState(null);
+    const [mapState, setMapState]: any = useState(null);
```
Comment (cholakovvv) [inline client/src/components/NavigationBar/Navigation.tsx:19]: is not a good practice to use type any in ts, when you hover over the element in this case setMapState it shows you the type

File: client/src/components/Styles/Header.scss [hunk 4]
```diff
@@ -0,0 +1,72 @@
+.top-line{
+    background-color: white;
+    width: 100%;
+    color: white;
+}
+.header-main-class{
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    background-color: rgb(235, 233, 233);
```
Comment (cholakovvv) [inline client/src/components/Styles/Header.scss:11]: this can be a external variable

**Review by cholakovvv — COMMENTED** [review]
I have some suggestions

File: client/src/components/Header.tsx (hunk 1 above)
Comment (IvayloSlavchev) [reply client/src/components/Header.tsx:12]: TextField and Box are two separated things. TextField is like input in HTML and Box is like a div, so I need to use Box(div) in this case.

**Review by di-amond — APPROVED** [review]
The task with curves was tricky element of the design and to be executed. But you did a great job making it!

**Review by petya0111 — APPROVED** [review]
Great work on your task and everything is on time! I am glad you have made it :)

**Pull request merged**